web/models.py

```
@@ -2930,6 +2930,37 @@
+            ("rejected", "Rejected"),
+        ],
+        default="pending",
+    )
+    fulfilled_by = models.ForeignKey(
+        EducationalVideo, on_delete=models.SET_NULL, related_name="fulfilling_requests", null=True, blank=True
+    )
+
+    class Meta:
+        verbose_name = "Video Request"
+        verbose_name_plural = "Video Requests"
+        ordering = ["-created_at"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)